CMakeLists.txt

```diff
@@ -10,8 +10,8 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 ###
 #SET("OpenCV_DIR" "~/apps/opencv-2.4.13/")
 set(PROJECT_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/include" CACHE PATH "Project Include Directory")
-set(SPINNAKER_INCLUDE_DIR "/usr/include/spinnaker" CACHE PATH "Spinnaker Include Directory")
-set(SPINNAKER_LIB_DIR "/usr/lib" CACHE PATH "Spinnaker Libs Directory")
+set(SPINNAKER_INCLUDE_DIR "/opt/spinnaker/include" CACHE PATH "Spinnaker Include Directory")
```

Comment:
This won't work.  This will break the code with older versions of spinnaker.  We need to have a way to let it still work with spinnaker 2.0 etc. as suggested in #105, we can to use find cmake OR add both the paths in there somehow.  @PushyamiKaveti  any ideas?

Reply:
Made the necessary changes to the CMakeLists.txt so that it works with both versions of spinnaker. This is fixed in branch hotfix/spinnaker_cmake